### PR TITLE
Secure-login-gov. LoginTest

### PR DESCRIPTION
**Pull request by FoXCity1337**
-add dependencies
-add suit for secure login gov
-add test to validate login form with empty data
-add test to validate login form with any email
-add test to validate login form with any password
-add test to validate login form with any data

### CODE REVIEW

File: src/test/java/gov/login/secure/LoginTest.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class LoginTest {
+
+    @Test
+    public void test1() {
+        WebDriver webDriver = new ChromeDriver();
+        webDriver.navigate().to("https://secure.login.gov/");
+        String xpath = "/html/body/main/div/form/lg-submit-button/button";
```
Comment (github4ta) [inline src/test/java/gov/login/secure/LoginTest.java:15]: xpath - не говорящее название

File: src/test/java/gov/login/secure/LoginTest.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        By by = new By.ByXPath(xpath);
```
Comment (github4ta) [inline src/test/java/gov/login/secure/LoginTest.java:16]: by - не говорящее название

File: src/test/java/gov/login/secure/LoginTest.java [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+        WebElement webElement = webDriver.findElement(by);
```
Comment (github4ta) [inline src/test/java/gov/login/secure/LoginTest.java:17]: webelement не говорящее название

**Comment by github4ta** [comment]
Нужно чтобы xpath, by, webelement стали говорящими названиями для каждого веб елемента своими индивидуальными

File: src/test/java/gov/login/secure/LoginTest.java (hunk 2 above)
Comment (FoXCity1337) [reply src/test/java/gov/login/secure/LoginTest.java:16]: Исправлено
